### PR TITLE
fix(core): add CLI instructions when localize polyfill is missing

### PR DESCRIPTION
**Pull request by vikerman**
Resolves FW-1627

### CODE REVIEW

File: integration/side-effects/snapshots/core/esm2015.js [hunk 1]
```diff
@@ -13,5 +13,5 @@ const __global = "undefined" !== typeof global && global;
 const _global = __globalThis || __global || __window || __self;
 
 if (ngDevMode) _global.$localize = _global.$localize || function() {
-    throw new Error("It looks like your application or one of its dependencies is using i18n.\n" + "Angular 9 introduced a global `$localize()` function that needs to be loaded.\n" + "Please add `import '@angular/localize/init';` to your polyfills.ts file.");
+    throw new Error("It looks like your application or one of its dependencies is using i18n.\n" + "Angular 9 introduced a global `$localize()` function that needs to be loaded.\n" + "Please run `ng add @angular/localize` from the Angular CLI.\n" + "(For non-CLI projects add `import '@angular/localize/init';` to your polyfills.ts file)");
```
Comment (kara) [inline integration/side-effects/snapshots/core/esm2015.js]: Grammar nit

```suggestion
    throw new Error("It looks like your application or one of its dependencies is using i18n.\n" + "Angular 9 introduced a global `$localize()` function that needs to be loaded.\n" + "Please run `ng add @angular/localize` from the Angular CLI.\n" + "(For non-CLI projects, add `import '@angular/localize/init';` to your polyfills.ts file)");
```

**Review by kara — APPROVED** [review]
LGTM aside from minor nit

**Review by IgorMinar — APPROVED** [review]
lgtm. thanks!

**Comment by matsko** [comment]
Presubmit = https://test.corp.google.com/ui#id=OCL:275356256:BASE:275356281:1571352793599:c07c1c9f

Ivy Presubmit = https://tap.corp.google.com/ui#id=OCL:275356256:BASE:275356609:1571352874030:71d3804a

**Pull request closed without merging**